ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/logger/TestChronicleAuditLogger.java

```diff
@@ -0,0 +1,159 @@
+ * limitations under the License.
+ */
+package com.ericsson.bss.cassandra.ecaudit.logger;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.time.Instant;
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ericsson.bss.cassandra.ecaudit.entry.AuditEntry;
+import com.ericsson.bss.cassandra.ecaudit.entry.SimpleAuditOperation;
+import com.ericsson.bss.cassandra.ecaudit.entry.Status;
+import net.openhft.chronicle.wire.ValueOut;
+import net.openhft.chronicle.wire.WireOut;
+import net.openhft.chronicle.wire.WriteMarshallable;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoMoreInteractions;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
+public class TestChronicleAuditLogger
+{
+    @Mock
+    private ChronicleWriter mockWriter;
+
+    @Mock
+    private WireOut mockWire;
+
+    @Mock
+    private ValueOut mockValue;
+
+    private ChronicleAuditLogger logger;
+
+    @Before
+    public void before()
+    {
+        logger = new ChronicleAuditLogger(mockWriter);
+    }
+
+    @After
+    public void after()
+    {
+        verifyNoMoreInteractions(mockWire);
+        verifyNoMoreInteractions(mockValue);
+    }
+
+    @Test
+    public void singleStatement() throws Exception
+    {
+        AuditEntry auditEntry = givenAuditEntry(Instant.parse("1993-07-27T18:15:30Z").toEpochMilli(), "Javier Sotomayor", "2.45.2.45", "High Jump", Status.ATTEMPT, null);
+
+        logger.log(auditEntry);
+
+        verifyWire(Instant.parse("1993-07-27T18:15:30Z").toEpochMilli(),"Javier Sotomayor", "2.45.2.45", "High Jump", Status.ATTEMPT, null);
+    }
+
+    @Test
+    public void batchStatement() throws Exception
+    {
+        AuditEntry auditEntry = givenAuditEntry(Instant.parse("1993-07-27T18:15:30Z").toEpochMilli(), "Javier Sotomayor", "2.45.2.45", "High Jump", Status.ATTEMPT, UUID.fromString("4910e9a6-9d26-40f8-ad8c-5c0436784969"));
+
+        logger.log(auditEntry);
+
+        verifyWire(Instant.parse("1993-07-27T18:15:30Z").toEpochMilli(),"Javier Sotomayor", "2.45.2.45", "High Jump", Status.ATTEMPT, UUID.fromString("4910e9a6-9d26-40f8-ad8c-5c0436784969"));
+    }
+
+    @Test
+    public void interruptOnPut() throws Exception
+    {
+        AuditEntry auditEntry = givenAuditEntry(Instant.parse("1993-07-27T18:15:30Z").toEpochMilli(), "Javier Sotomayor", "2.45.2.45", "High Jump", Status.ATTEMPT, UUID.fromString("4910e9a6-9d26-40f8-ad8c-5c0436784969"));
```

Comment:
We don't seem to have any Status.FAILED? Might not make a difference from a test perspective though?